test/staking/rebate.test.ts

```diff
@@ -14,6 +16,53 @@ const toFixed = (n: number | BigNumber, precision = 12) => {
   return toFloat(n).toFixed(precision)
 }
 
+const ABI_LIB_EXPONENTIAL = [
```

Comment:
Same as above, could this use hardhat artifacts?

Reply:
https://github.com/graphprotocol/contracts/issues/887 😄